lib/esp-scanning-lib.robot

```diff
@@ -0,0 +1,126 @@
+*** Settings ***
+Documentation       Collection of keywords related to UEFI Secure Boot
+
+Resource            ../keywords.robot
+
+
+*** Variables ***
+${SHIMX64_URL}=     https://cloud.3mdeb.com/index.php/s/TipPHgsCj6MCZHi/download/shimx64.efi
+${GRUBX64_URL}=     https://cloud.3mdeb.com/index.php/s/e2JjYANjMsgyXtS/download/grubx64.efi
+
+
+*** Keywords ***
+Populate /EFI With System Folders
+    [Documentation]    Adds all supported system folders and files to /EFI
+    # given that most of our machines have ubuntu installed,
+    # it's best not to mess with ubuntu EFI files in any way.
+    Add System Folder To /EFI    linux    Fedora
+    Add System Folder To /EFI    manager    Suse
+    Add System Folder To /EFI    windows
+    Add System Folder To /EFI    manager    Redhat
+    Add System Folder To /EFI    linux    Redhat
+    Add System Folder To /EFI    linux    Centos
+    Add System Folder To /EFI    linux    opensuse
+    Add System Folder To /EFI    linux    qubes
+    Add System Folder To /EFI    linux    DTS
+    Add System Folder To /EFI    linux    debian
+
+Add System Folder To /EFI
+    [Documentation]    This keyword creates directories and .efi files for
+    ...    given systems. Available options:
+    ...    ${type} - linux, windows, manager
```

Comment:
What `manager` stands for?

Reply:
It's to handle the boot managers - they use elilo.efi instead of grubx64/shimx64. Will clarify that more in the keyword's documentation